README.md

```diff
@@ -1,44 +1,128 @@
 # Directed Acyclic Workflow Graph Scheduling
 
+Reproducible research and reusable acyclic workflows in Python. Execute code on HPC systems as if you executed them on your machine!
+
+## Motivation
+
+Would you like fully reproducible research or reusable workflows that seamlessly run on HPC clusters?
+Tired of writing and managing large Slurm submission scripts? Do you have comment out large parts of your pipeline whenever its results have been generated? Hate YAML?
+Don't waste your precious time! `dawgz` allows you to directly describe complex pipelines in Python, that run on your personal computer and large HPC clusters.
+
+
```

Comment:
## Installation

pip install and all

## Getting started

only the experiment with pi (others in exemples folder)

## Roadmap

## Contributing

* issues blabla
* discussions